04_pig_latin/pig_latin.rb

```diff
@@ -0,0 +1,33 @@
+$vow = ['a', 'e', 'i', 'o', 'u']
+$con = ['b', 'c', 'd', 'f', 'g', 'h', 'j', 'k', 'l', 'm', 'n', 'p', 'q', 'r', 's', 't', 'w', 'v', 'x', 'y', 'z']
+
+def translate(str)
+  str.split.map { |curr_word|  translate_word(curr_word) }.join(' ')
+end
+
+def translate_word(word)
+  final_word = ''
+  word[0] = word[0].downcase
+
+  if $vow.include? word[0]
+    final_word += translate_vow(word)
```

Comment:
I think the `final_word` variable isn't needed - try to find a solution that doesn't use it.